--- plugged/coc.nvim/autoload/coc/util.vim
+++ plugged/coc.nvim/autoload/coc/util.vim
@@ -462,12 +462,13 @@ function! coc#util#do_complete(name, opt, cb) abort
 endfunction
 
 function! coc#util#extension_root() abort
-  if s:is_win
-    let dir = $HOME.'/AppData/Local/coc/extensions'
-  else
-    let dir = $HOME.'/.config/coc/extensions'
-  endif
-  return dir
+  return coc#util#get_config_home().'/coc/extensions'
+  "if s:is_win
+    "let dir = $HOME.'/AppData/Local/coc/extensions'
+  "else
+    "let dir = $HOME.'/.config/coc/extensions'
+  "endif
+  "return dir
 endfunction
 
 function! coc#util#install_extension(names) abort
